share/completions/adb.fish

```diff
@@ -58,6 +58,21 @@ function __fish_adb_list_uninstallable_packages
     __fish_adb_run_command pm list packages -3 | string replace 'package:' ''
 end
 
+function __fish_adb_list_files
+   set -l token (commandline -ct)
+
+   # Have tab complete show initial / if nothing on current token
+   if test -z "$token"
+       set token "/"
+   end
+
+   # Return list of directories suffixed with '/'
+   __fish_adb_run_command find -H "$token*" -maxdepth 0 -type d 2>/dev/null | awk '{print $1"/"}'
```

Comment:
It's actually possible to do this with one find invocation.

Please try

```fish
__fish_adb_run_command find -H "$token*" -maxdepth 0 \( -type d -printf "%p/\n" , -type f -print \) 2>/dev/null
```

Reply:
adb shell's find command doesn't support -printf. Just tried it :(
```
find: bad arg '-printf'
```

Reply:
Ah, that's that minimalism everyone is so enthusiastic about? :anger:

I've merged this now, thanks!